documents/operating.md

```diff
@@ -17,7 +17,9 @@ SECTION 0: 📓📓📓 Definitions 📓📓📓
 
 * **Bad Idea Factory Limited Liability Company ("Company")** is a Limited Liability Company domiciled in Pennsylvania.
 * Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
-* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* "**Corporate Overlord With Status (COWS)**" shall mean an "owner" of the Company under Pennsylvania state law.
```

Comment:
"Status" seems like an airline perks level thing

Reply:
To follow up from this comment I can't help but feel that there's a weird "separate but equal" distinction that's baked into how we're naming these groups. We probably can't get away from the separation because of the legal status at play, but I think the spirit of what we're trying to do is to prevent this from turning out like two tiers of Corporate Overlord status.

Like, we should probably define "Corporate Overlord" to be what it is now (voting rights, profit sharing), _except_ that Corporate Overlords have nothing to do with legal ownership of the company.

Separately, _some_ Corporate Overlords (or, as many as are able -- those that will not be burdened by corporate ownership) take on the additional responsibility of acting a legal owner for the purposes of US law. This is purely a ceremonial role, as we've learned that you can be a legal owner without doing much of anything else. However, this might impact how one pays a CO and how that CO reports taxes; this may also impact things like which COs can be involved in certain things, like open bank accounts.

Reply:
One note re the last thing: right now only the overmind has legal authority to enter contracts (unless the overmind gives a green light to a contract being negotiated by another CO, which may not actually be ok according to bylaws but we've done it anyway)

The intent of this language is I think what you are asking for lou -- I was just saying "Corporate Overlords are the union of these two groups of people, and the only thing that you get from being a COWS is legal ownership."

It sounds like maybe the word status is what is flagging concern -- happy to pick another name.  "Corporate Overlords Who Are Also Owners" but that acronym stinks.